vision/google/cloud/vision/annotations.py

```diff
@@ -87,6 +87,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.annotations.Annotations`
         :returns: An instance of ``Annotations`` with detection types loaded.
         """
+        for feature_type in response.keys():
```

Comment:
Not sure I really like this. But it seemed like the most direct strategy. Let me know if you have a preferred way to do this.